### ISSUE TITLE
GPT-3 prefers certain multiple choice options

### ISSUE THREAD

**Issue by aogara-ds** [aogara-ds/hoodwinked-website#5]
Pretrained language models are known to prefer certain kinds of answers for spurious reasons. [Zhao 2021 ](https://arxiv.org/pdf/2102.09690.pdf) shows that GPT-3 prefers certain answer choices to multiple choice questions when the question is "N/A". Fixing this miscalibration can strongly improve performance in multiple choice QA. 

Our GPT-3 action agent suffers from the same problem. When the killer is given seven answer choices, the last four of which involve killing someone, it chooses answer number 4 with unreasonably high probability. 

<img width="524" alt="image" src="https://user-images.githubusercontent.com/78378219/194665028-a5945e60-683a-4557-917a-6ad3c63fe774.png">

To solve this, we could fine-tune GPT-3 with the calibration technique proposed in Zhao 2021, or we could just run RL training.

**Comment by iejMac** [comment]
I think you should leave fine-tuning as a last resort, try some other prompting methods, maybe even shuffle.

Is it possible that it's just choosing the first "kill" option? It's clearly stated that killing is it's objective and option 4 is the first among 4 that are all equivalent.
